### PR TITLE
Feature/image loading type and default size

### PR DESCRIPTION
**Pull request by waldemar-p**
## Summary

I am working on the loading performance and therefore I added two properties to the `image-widget`.
Via `loadingType` the attribute `loading={{loadingType}}` can be set.
And the `size` can now be set in the `manager.options` to have a default size other than `'full'`.

## What are the specific steps to test this change?

1. Create a file `sites/modules/@apostrophecms/image-widget/index.js` with 
```javascript
module.exports = {
  options: {
    size: 'one-sixth',
    loadingType: 'lazy'
  },
};
```
2. Upload an image and use it on the website
3. Check the HTML of it.
4. The `img`-tag should have `loading="lazy"` and the `src` should have the image with the size of `one-sixth`

## What kind of change does this PR introduce?

- [ ] Bug fix
- [X] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [x] It includes **a) the existing issue ID being resolved**, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves (at least I hope it is convincing)
- [x] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

Issue (I just opened): #4020

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@
 * Basic support for editing tables by adding `table` to the rich text toolbar. Enabling `table` allows you to create tables, including `td` and `th` tags, with the ability to merge and split cells. For now the table editing UI is basic, all of the functionality is there but we plan to add more conveniences for easy table editing soon. See the "Table" dropdown for actions that are permitted based on the current selection.
 * Support for `div` tags in the rich text toolbar, if you choose to include them in `styles`. This is often necessary for A2 content migration and can potentially be useful in new work when combined with a `class` if there is no suitable semantic block tag.
 * The new `@apostrophecms/attachment:download-all --to=folder` command line task is useful to download all of your attachments from an uploadfs backend other than local storage, especially if you do not have a more powerful "sync" utility for that particular backend.
+* For the widget `image-widget`
```
Comment (BoDonkey) [inline CHANGELOG.md:11]: Can you combine the first two lines into a single sentence, please? Something like, "Two new options have been added to the `image-widget`: `loadingType` and `size`."

File: modules/@apostrophecms/rich-text-widget/index.js [hunk 2]
```diff
@@ -323,7 +323,7 @@ module.exports = {
           underline: [ 'u' ],
           anchor: [ 'span' ],
           superscript: [ 'sup' ],
-          subscript: [ 'sub' ]
+          subscript: [ 'sub' ],
```
Comment (BoDonkey) [inline modules/@apostrophecms/rich-text-widget/index.js:326]: Would it be possible for you to remove this commit from your PR? Somehow you have some of boutell's commits mixed in with your PR.

**Review by BoDonkey — CHANGES_REQUESTED** [review]
This looks pretty good. Just a small change to the README and a clean-up of your PR. If it is too difficult to pull the extra commits out of this PR, can you create a new clean PR? Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (waldemar-p) [reply CHANGELOG.md:11]: Is the wording now better?

Comment (BoDonkey) [reply CHANGELOG.md:11]: Yup, looks good to me.

File: modules/@apostrophecms/image-widget/views/widget.html [hunk 3]
```diff
@@ -5,24 +5,18 @@
...
-  {% endif %}
-
-  {% if data.options.dimensionAttrs %}
-    {% set dimensionAttrs = data.options.dimensionAttrs %}
-  {% elif data.manager.options.dimensionAttrs %}
-    {% set dimensionAttrs = data.manager.options.dimensionAttrs  %}
-  {% endif %}
+  {% set className = data.options.className or data.manager.options.className %}
+  {% set dimensionAttrs = data.options.dimensionAttrs or data.manager.options.dimensionAttrs %}
+  {% set loadingType = data.options.loadingType or data.manager.options.loadingType %}
```
Comment (boutell) [inline modules/@apostrophecms/image-widget/views/widget.html:10]: Very nice, having two levels of fallback for both of these 👍

**Review by boutell — APPROVED** [review]
Looks good to me. Bob can you add a docs PR that covers these and also explains the function of `loading`? Looks like it falls under "cool features not all browsers have yet 100% but which don't hurt if they are missing":

https://caniuse.com/loading-lazy-attr

**Review by BoDonkey — APPROVED** [review]
Everything looks good. Thanks for the changes! I'll be merging this now and it should be released next Wednesday.

**Pull request merged**